src/com/machinepublishers/jbrowserdriver/Settings.java

```diff
@@ -1158,6 +1158,7 @@ public Capabilities buildCapabilities() {
       set(capabilities, PropertyName.CONNECTION_REQ_TIMEOUT_MS, this.connectionReqTimeout);
       //TODO set(capabilities, PropertyName.RESPONSE_INTERCEPTORS, this.responseInterceptors);
       set(capabilities, PropertyName.JAVA_OPTIONS, StringUtils.join(this.javaOptions, "\t"));
+      set(capabilities, PropertyName.USER_DATA_DIRECTORY, this.userDataDirectory.toString());
```

Comment:
Possible NPE? Also it might be better to get the absolute path instead of `toString`